balances/src/mock.rs

```diff
@@ -52,8 +57,10 @@ impl_encointer_scheduler!(TestRuntime);
 // genesis values
 pub fn new_test_ext() -> sp_io::TestExternalities {
 	let mut t = frame_system::GenesisConfig::default().build_storage::<TestRuntime>().unwrap();
-	dut::GenesisConfig { demurrage_per_block_default: Demurrage::from_bits(DEFAULT_DEMURRAGE) }
-		.assimilate_storage(&mut t)
-		.unwrap();
+	dut::GenesisConfig {
+		demurrage_per_block_default: Demurrage::from_bits(DefaultDemurrage::get()),
```

Comment:
If we have that as a parameter type, we don't need the genesis config value anymore.

Reply:
removing line 60 results in a failed tests:
tests::transfer_with_demurrage_exceeding_amount_should_fail
tests::demurrage_should_work

Reply:
Sorry for not being clear. Consequently, the genesis value should be removed from the `EncointerBalances` pallet completely and all calls within the pallet that access the genesis field value should fetch the parameter type genesis value.